.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish Package to npmjs
+on:
+    release:
```

Comment:
can you add instructions to the README describing the process? See https://github.com/vimc/montagu-deploy?tab=readme-ov-file#publishing-to-pypi for an example, others exist on other python versions too